scripts/local/versions.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Set up the versions to be used
+awm_relayer_version=${AWM_RELAYER_VERSION:-'v0.2.1'}
```

Comment:
can't we uppercase these similar to `GINKGO_VERSION`? We'd still be able to override when set.